WORKSPACE

```diff
@@ -97,9 +97,6 @@ buildifier_prebuilt_register_toolchains()
 load(
     "@aspect_rules_lint//format:repositories.bzl",
     "fetch_shfmt",
-    "fetch_terraform",
```

Comment:
Because this repo doesn't use terraform?

Reply:
Yes. There used to be some terraform in here for Workflows examples but that is now removed.